src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -544,47 +537,54 @@ struct
         breadcrumb |> Breadcrumb.staged_ledger
         |> Staged_ledger.pending_coinbase_collection
 
-      let handle_diff () (diff : Breadcrumb.t Transition_frontier_diff.t) :
-          view option =
-        let open Transition_frontier_diff in
-        let open Diff_mutant.E in
-        Option.return
-        @@
-        match diff with
-        | New_frontier breadcrumb ->
-            [ E
-                (New_frontier
-                   ( Breadcrumb.transition_with_hash breadcrumb
-                   , scan_state breadcrumb
-                   , pending_coinbase breadcrumb )) ]
-        | New_breadcrumb breadcrumb ->
-            [E (Add_transition (Breadcrumb.transition_with_hash breadcrumb))]
-        | New_best_tip {garbage; added_to_best_tip_path; new_root; old_root; _}
-          ->
+      let consensus_state breadcrumb =
+        breadcrumb |> Breadcrumb.transition_with_hash |> With_hash.data
+        |> External_transition.Validated.protocol_state
+        |> Coda_state.Protocol_state.consensus_state
+
+      let get_root_data root =
+        (Breadcrumb.state_hash root, scan_state root, pending_coinbase root)
+
+      let compute_ground_truth_mutants = function
```

Comment:
Can you leave a comment explaining this function? What is a "ground_truth_mutant"?